### PR TITLE
added the ingredients modal

### PR DESCRIPTION
**Pull request by bhaumikdebanshu**

### CODE REVIEW

**Review by ashishsfb — COMMENTED** [review]
I think it should be ok. Ingredient modal

File: public/product/modal.css [hunk 1]
```diff
@@ -14,11 +14,18 @@
   border-radius: 5px;
   box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.25);
   background-color: #fff;
-  margin-top: 300px;
+  margin-top: auto;
```
Comment (ashishsfb) [inline public/product/modal.css:17]: Use align-items: center; on modal-container

File: public/product/modal.css [hunk 2]
```diff
@@ -14,11 +14,18 @@
...
 
 .modal.modal-review {
   width: 340px;
   height: 300px;
   max-height: 400px;
 }
+
+.modal.modal-ingredients {
+  width: 340px;
+  /* height: 300px; */
```
Comment (ashishsfb) [inline public/product/modal.css:29]: Remove comment

**Review by ashishsfb — CHANGES_REQUESTED** [review]
Requesting changes

File: public/product/modal.css (hunk 2 above)
Comment (bhaumikdebanshu) [reply public/product/modal.css:29]: done

**Review by ashishsfb — APPROVED** [review]
Approved changes on ingredient modal

**Pull request merged**